charts/backstage/templates/_helpers.tpl

```diff
@@ -21,3 +21,29 @@ Returns a secret name for service to service auth
         {{- include "common.names.fullname" . -}}-auth
     {{- end -}}
 {{- end -}}
+
+{{/*
+Sets the secretKeyRef name for Backstage to the PostgreSQL existing secret if it present
+*/}}
+{{- define "janus-idp.backstage.existingSecret" -}}
+    {{- if .Values.postgresql.auth.existingSecret -}}
+        {{- .Values.postgresql.auth.existingSecret -}}
+    {{- else if .Values.global.postgresql.auth.existingSecret -}}
+        {{- .Values.global.postgresql.auth.existingSecret -}}
+    {{- else -}}
+        {{- printf "%s-%s" .Release.Name "postgresql" -}}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Sets the secretKeyRef name for PostgreSQL to the PostgreSQL existing secret if it present
+*/}}
+{{- define "janus-idp.postgresql.existingSecret" -}}
```

Comment:
Instead of this, can't we reuse (in this case copypaste in here:

https://github.com/bitnami/charts/blob/532b73c06f9f554f2f8a971226df5b9e44707df9/bitnami/postgresql/templates/_helpers.tpl#L94-L105

and reference it as `{{ include "postgresql.v1.secretName" . }}` instead? I think it would be cleaner to instead of declaring "hey, we have a different template", we just said "we just want the same behavior". WDYT?